packages/icon/src/Icon.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React, { PureComponent } from 'react';
+import { createStyledComponent, generateId } from '@mineral-ui/component-utils';
+
+type Props = {
+  /** Available sizes, including custom - e.g. '5em' or '20px' */
+  size?: 'small' | 'medium' | 'large' | string,
+  /** SVG content, required for the generic Icon component */
+  children?: MnrlReactNode,
+  /** Fill color, defaults to theme.color_gray_60 */
+  color?: string,
+  /** Flip the icon horizontally when used with RTL languages */
+  rtlMirror?: boolean,
```

Comment:
that might be good in case setting `rtl` ever does anything besides simply flipping the component visually. I can't imagine what that would be, but you never know.